src/sentry/search/events/fields.py

```diff
@@ -2265,13 +2275,63 @@ def __init__(self, dataset: Dataset, params: ParamsType):
                     ),
                     default_result_type="percentage",
                 ),
+                SnQLFunction(
+                    "percentile",
+                    required_args=[
+                        SnQLColumnOrAlias("column"),
+                        NumberRange("percentile", 0, 1),
+                    ],
+                    snql_aggregate=self._resolve_percentile,
+                    result_type_fn=reflective_result_type(),
+                    default_result_type="duration",
+                    redundant_grouping=True,
+                ),
+                SnQLFunction(
+                    "p50",
+                    optional_args=[
+                        with_default("transaction.duration", SnQLColumnOrAlias("column")),
+                    ],
+                    snql_aggregate=lambda args, alias: self._resolve_percentile(args, alias, 0.5),
+                    default_result_type="integer",
```

Comment:
if your default column is going to be `transaction.duration` the default result type should probably be `duration`

Reply:
Fixed!